ClosedXML.Tests/Excel/CalcEngine/MathTrigTests.cs

```diff
@@ -9,6 +9,7 @@
 namespace ClosedXML.Tests.Excel.CalcEngine
 {
     [TestFixture]
+    [SetCulture("")]
```

Comment:
This is problematic, because what it essentially does is mask potential problems when culture of user is different than invariant culture. It's desirable that tests are run in many cultures and still work correctly, after all ClosedXml is also run with different current cultures.

The problem with failing test is that interpolated string formats MROUND formula with a current culture and passes it as an input to CalcEngine. By formatting it in current culture, it sends invalid data (valid formula has to satisfy the grammar).

Correct fix would be to create a correct input for CalcEngine, something like

```
string.Format(CultureInfo.InvariantCulture, "MROUND({0}, {1})", number, multiple)
```